commandline/src/main/java/tech/pegasys/eth2signer/commandline/VersionInfo.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.commandline;
+
+import picocli.CommandLine.IVersionProvider;
+
+public class VersionInfo implements IVersionProvider {
+
+  @Override
+  public String[] getVersion() throws Exception {
+    return new String[] {"Version 0.0.1"};
```

Comment:
I am assuming this will be injected eventually come from build script?

Reply:
This is the same as ethsigner ... I _suspect_ its gets overridden with gradle properties?

Reply:
hmm...lets leave it as it is, we can re-visit it again, its not a show-stopper at the moment.